smsframework_africastalking/error.py

```diff
@@ -0,0 +1,6 @@
+from smsframework.exc import ProviderError, RequestError
+
+class AfricasTalkingProviderError(ProviderError):
+    
+    def __init__(self, message):
+        super().__init__(message)
```

Comment:
you need to press enter and add one more empty line(PEP-8) :)

Reply:
done